src/components/Contexts/OrganizationListContext.test.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { FC } from "react";
+import { render, waitFor } from "@testing-library/react";
+import { MockedProvider, MockedResponse } from "@apollo/client/testing";
+import { GraphQLError } from "graphql";
+import {
+  OrganizationProvider,
+  Status as OrgStatus,
+  useOrganizationListContext,
+} from "./OrganizationListContext";
+import { LIST_ORGS } from "../../graphql";
+
+type Props = {
+  mocks?: MockedResponse[];
+  children?: React.ReactNode;
+};
+
+const TestChild: FC = () => {
+  const { status, data } = useOrganizationListContext();
+
+  if (status === OrgStatus.LOADING) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div>
+      <div data-testid="status">{status}</div>
+      <ul data-testid="organization-list">
+        {data?.map((org, index) => (
+          <li key={org?._id} data-testid={`organization-${index}`}>
+            {org.name}
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+const TestParent: FC<Props> = ({ mocks, children }: Props) => (
+  <MockedProvider mocks={mocks} addTypename={false}>
+    <OrganizationProvider filterInactive={false} preload>
+      {children ?? <TestChild />}
+    </OrganizationProvider>
+  </MockedProvider>
+);
+
+describe("OrganizationListContext > useOrganizationListContext Tests", () => {
+  it("should throw an exception when used outside of the OrganizationProvider", () => {
+    jest.spyOn(console, "error").mockImplementation(() => {});
+    expect(() => render(<TestChild />)).toThrow(
+      "OrganizationListContext cannot be used outside of the OrganizationProvider component"
+    );
+    jest.spyOn(console, "error").mockRestore();
+  });
+});
+
+describe("OrganizationListContext > OrganizationProvider Tests", () => {
+  it("should render without crashing", () => {
+    render(<TestParent />);
```

Comment:
Apollo is reporting missing mocks. It doesn't say which test is causing it, but I think it's this one and maybe the following test too?

![Screenshot 2024-06-06 at 12 13 03 PM](https://github.com/CBIIT/crdc-datahub-ui/assets/38357871/9d2d7200-eb07-41ba-b02c-4e96571e5f24)